src/kafka-file/kafkaFileClient.ts

```diff
@@ -5,13 +5,15 @@ import { ClusterSettings } from "../settings/clusters";
 
 import { getLanguageModelCache, LanguageModelCache } from './languageModelCache';
 import { KafkaFileDocument } from "./languageservice/parser/kafkaFileParser";
-import { ConsumerLaunchStateProvider, getLanguageService, LanguageService, ProducerLaunchStateProvider, SelectedClusterProvider } from "./languageservice/kafkaFileLanguageService";
+import { ConsumerLaunchStateProvider, getLanguageService, LanguageService, ProducerLaunchStateProvider, SelectedClusterProvider, TopicDetail, TopicProvider } from "./languageservice/kafkaFileLanguageService";
 import { runSafeAsync } from "./utils/runner";
+import { KafkaExplorer, TopicItem } from "../explorer";
 
 export function startLanguageClient(
     clusterSettings: ClusterSettings,
     producerCollection: ProducerCollection,
     consumerCollection: ConsumerCollection,
+    explorer: KafkaExplorer,

```

Comment:
don't inject the KafkaExplorer, which is too UI oriented, but a KafkaModelProvider instance instead

Reply:
Good catch! Fixed.